Mathlib/Order/Interval/Set/SuccOrder.lean

```diff
@@ -26,11 +26,27 @@ lemma Iic.succ_coe_of_not_isMax
   apply Order.succ_le_of_lt
   exact lt_of_le_of_ne (α := Set.Iic j) le_top (by simpa using hi)
 
+lemma Iic.succ_eq_of_not_isMax
```

Comment:
Is it possible to prove this first? That would make the other lemma a simple `rw`.

Reply:
No, the statement itself uses the first version of the lemma.